### PR TITLE
[Communication] Phone Number - Update Swagger

### PR DESCRIPTION
**Pull request by minnieliu**

### CODE REVIEW

**Review by sacheu — APPROVED** [review]

File: sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/implementation/models/package-info.java [hunk 1]
```diff
@@ -4,6 +4,6 @@
 
 /**
  * Package containing the data models for PhoneNumberAdminClient. The phone numbers client uses Azure Communication
- * Services to acquire and manage phone numbers.
```
Comment (sacheu) [inline sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/implementation/models/package-info.java:7]: nit. The class name is `PhoneNumberAdminClient` but the comment says `The phone number**s** client`

File: sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/implementation/package-info.java [hunk 2]
```diff
@@ -4,6 +4,6 @@
 
 /**
  * Package containing the implementations and inner classes for PhoneNumberAdminClient. The phone numbers client uses
- * Azure Communication Services to acquire and manage phone numbers.
```
Comment (sacheu) [inline sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/implementation/package-info.java:7]: nit. The class name is PhoneNumberAdminClient but the comment says The phone number**s** client

File: sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/models/package-info.java [hunk 3]
```diff
@@ -4,6 +4,6 @@
 
 /**
  * Package containing classes for PhoneNumberAdminClient. The phone numbers client uses Azure Communication Services to
- * acquire and manage phone numbers.
```
Comment (sacheu) [inline sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/models/package-info.java:7]: nit. The class name is PhoneNumberAdminClient but the comment says The phone number**s** client

File: sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/implementation/models/package-info.java (hunk 1 above)
Comment (minnieliu) [reply sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/implementation/models/package-info.java:7]: This is generated code so we should avoid editing manually

**Pull request merged**